lib/features/home/presentation/view_model/home_view_model.dart

```diff
@@ -14,14 +15,21 @@ class HomeViewModel extends BaseViewModel {
   final ValueNotifier<List<Product>> _arrivalsNotifier = ValueNotifier([]);
   ValueNotifier<List<Product>> get arrivalsNotifier => _arrivalsNotifier;
 
+  final ValueNotifier<List<Category>> _categoriesNotifier = ValueNotifier([]);
+  ValueNotifier<List<Category>> get categoriesNotifier => _categoriesNotifier;
+
   final ValueNotifier<bool> _popularProductsloading = ValueNotifier(false);
   ValueNotifier<bool> get popularProductsloading => _popularProductsloading;
 
   final ValueNotifier<bool> _newArrivalsLoading = ValueNotifier(false);
   ValueNotifier<bool> get newArrivalsLoading => _newArrivalsLoading;
 
+  final ValueNotifier<bool> _categoriesLoading = ValueNotifier(false);
+  ValueNotifier<bool> get categoriesLoading => _categoriesLoading;
+
   bool get isLoadingArrivals => _newArrivalsLoading.value;
   bool get isLoadingProducts => _popularProductsloading.value;
+  bool get isLoadingCategories => _categoriesLoading.value;
```

Comment:
The getters `isLoadingArrivals`, `isLoadingProducts` and `isLoadingCategories` are never used.
Remove unused code